test/gnmi/devicestatetest.go

```diff
@@ -15,23 +15,25 @@
 package gnmi
 
 import (
+	"testing"
+	"time"
+
 	"github.com/onosproject/onos-config/test/utils/gnmi"
 	"github.com/onosproject/onos-topo/api/device"
+	"github.com/onosproject/onos-topo/api/topo"
 	"github.com/stretchr/testify/assert"
-	"testing"
-	"time"
 )
 
 // TestDeviceState tests that a device is connected and available.
 func (s *TestSuite) TestDeviceState(t *testing.T) {
```

Comment:
This is an important test which confirms that onos-config can be connected to the devices.